include/boost/geometry/extensions/gis/io/wkb/detail/ogc.hpp

```diff
@@ -107,99 +107,61 @@ struct ewkt_policy
 {
 };
 
-template
-<
-    typename Geometry, 
-    typename CheckPolicy = ogc_policy, 
-    typename Tag = typename tag<Geometry>::type
->
-struct geometry_type : not_implemented<Tag>
-{
-};
-
-template <typename Geometry, typename CheckPolicy>
-struct geometry_type<Geometry, CheckPolicy, point_tag>
+template <typename Geometry,
+          geometry_type_ogc::enum_t OgcType,
```

Comment:
I know it is not part of this review (therefore I wrote "not introduced here"). So yes, it is a remark to the original author (probably Mateusz or Mats).
OK, inspecting the code it is probably part of the scoped enumerations from Boost. You can ignore this remark.